python/hail/methods/statgen.py

```diff
@@ -13,41 +13,51 @@
-    - **va.linreg.ytx** (*Array[Double]*) -- array of dot products of each response vector ``y`` with the input vector ``x``
-    - **va.linreg.beta** (*Array[Double]*) -- array of fit effect coefficients, :math:`\hat\beta_1`
-    - **va.linreg.se** (*Array[Double]*) -- array of estimated standard errors, :math:`\widehat{\mathrm{se}}`
-    - **va.linreg.tstat** (*Array[Double]*) -- array of :math:`t`-statistics, equal to :math:`\hat\beta_1 / \widehat{\mathrm{se}}`
-    - **va.linreg.pval** (*Array[Double]*) -- array of :math:`p`-values
-
-    :param ys: list of one or more response expressions.
-    :type ys: list of str
-
-    :param str x: expression for input variable
-
-    :param covariates: list of covariate expressions.
-    :type covariates: list of str
-
-    :param str root: Variant annotation path to store result of linear regression.
-
-    :param int variant_block_size: Number of variant regressions to perform simultaneously.  Larger block size requires more memmory.
-
-    :return: Variant dataset with linear regression variant annotations.
-    :rtype: :py:class:`.VariantDataset`
-
+    """Test each row for association with response variables using linear regression.
+
+    Examples
+    --------
+
+    >>> dataset_result = methods.linreg(dataset, [dataset.pheno.height], dataset.GT.num_alt_alleles(),
+    ...                                 covariates=[dataset.pheno.age, dataset.pheno.isFemale])
+
+    Warning
+    -------
+    :meth:`linreg` uses the same set of columns for each phenotype, the set of
+    columns for which **all** response variables and covariates are defined.
+
+    Notes
+    -----
+
+    With the default root, the following row-indexed fields are added.
+    The indexing of the array annotations corresponds to that of ``ys``.
+
+    - **linreg.nCompleteSamples** (*Int32*) -- number of columns used
+    - **linreg.AC** (*Float64*) -- sum of input values ``x``
+    - **linreg.ytx** (*Array[Float64]*) -- array of dot products of each response vector ``y`` with the input vector ``x``
+    - **linreg.beta** (*Array[Float64]*) -- array of fit effect coefficients, :math:`\hat\beta_1`
+    - **linreg.se** (*Array[Float64]*) -- array of estimated standard errors, :math:`\widehat{\mathrm{se}}`
+    - **linreg.tstat** (*Array[Float64]*) -- array of :math:`t`-statistics, equal to :math:`\hat\beta_1 / \widehat{\mathrm{se}}`
+    - **linreg.pval** (*Array[Float64]*) -- array of :math:`p`-values
+
+    Parameters
+    ----------
+    ys : :obj:`list` of :class:`hail.expr.expression.Expression`
+        One or more response expressions.
+    x : :class:`hail.expr.expression.Expression`
+        Input variable.
+    covariates : :obj:`list` of :class:`hail.expr.expression.Expression`
+        Covariate expressions.
+    root : :obj:`str`
+        Name of resulting row-indexed field.
+    block_size : :obj:`int`
+        Number of row regressions to perform simultaneously. Larger blocks
```

Comment:
I think it's clearer to say "simultaneously per core".

Reply:
make that fix in your PR, not worth retesting!